src/components/BookCard.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Great job working on this milestone. I don't think you should disable eslint, rather you should fix the errors that arise as the linters are there to make your robust and professional.